dockerfiles/sfpowerscripts.Dockerfile

```diff
@@ -65,23 +70,27 @@ RUN export XDG_DATA_HOME && \
     export XDG_CONFIG_HOME && \
     export XDG_CACHE_HOME
 
+# Install Yarn
+RUN npm install --global yarn
 
+# Install sfdx-cli
+RUN yarn global add sfdx-cli@7.145.0 
 
+# Install vlocity
+RUN yarn global add vlocity@1.15.2
 
-ENV PATH=/usr/local/lib/nodejs/bin:$PATH
-RUN npm install --global sfdx-cli@7.145.0 --ignore-scripts
-RUN  npm -g install vlocity@1.15.2
-
-
-ENV SFDX_CONTAINER_MODE true
+#Install Puppeteer
+RUN yarn global add puppeteer@10.4.0
 
+# Install all shared dependencies for chrome and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update && apt-get install -qq gconf-service libappindicator1 libasound2 libatk1.0-0 \
```

Comment:
Should cleanup apt cache after installation to keep image size small

Reply:
Should reorder as the dependencies don't change often

Reply:
had issues with pupetteer not recognizing these libs if installed earlier,  will need to investigate further